apps/dashboard/src/lib/components/Org/Settings/CustomizeLms.svelte

```diff
@@ -0,0 +1,208 @@
+        <Toggle size="sm" class="mb-3" bind:toggled={$currentOrg.customization.course.newsfeed}>
+          <span slot="labelA" style="color: gray"
+            >{$t('components.settings.customize_lms.disabled')}</span
+          >
+          <span slot="labelB" style="color: gray"
+            >{$t('components.settings.customize_lms.enabled')}</span
+          >
+        </Toggle>
+      </div>
+
+      <div>
+        <p>{$t('components.settings.customize_lms.course.grading')}</p>
+        <Toggle size="sm" class="mb-3" bind:toggled={$currentOrg.customization.course.grading}>
+          <span slot="labelA" style="color: gray"
+            >{$t('components.settings.customize_lms.disabled')}</span
+          >
+          <span slot="labelB" style="color: gray"
+            >{$t('components.settings.customize_lms.enabled')}</span
+          >
+        </Toggle>
+      </div>
+    </Column>
+  </Row>
+
+  <Row class="flex lg:flex-row flex-col py-7 border-bottom-c">
+    <Column sm={4} md={4} lg={4}
+      ><SectionTitle>{$t('components.settings.customize_lms.apps.title')}</SectionTitle></Column
+    >
+    <Column sm={8} md={8} lg={8} class="mt-4 lg:mt-0">
+      <div>
+        <p>{$t('components.settings.customize_lms.apps.poll')}</p>
+        <Toggle size="sm" class="mb-3" bind:toggled={$currentOrg.customization.apps.poll}>
+          <span slot="labelA" style="color: gray"
+            >{$t('components.settings.customize_lms.disabled')}</span
+          >
+          <span slot="labelB" style="color: gray"
+            >{$t('components.settings.customize_lms.enabled')}</span
+          >
+        </Toggle>
+      </div>
+      <div>
+        <p>{$t('components.settings.customize_lms.apps.live_comment')}</p>
+        <Toggle size="sm" class="mb-3" bind:toggled={$currentOrg.customization.apps.comments}>
+          <span slot="labelA" style="color: gray"
+            >{$t('components.settings.customize_lms.disabled')}</span
+          >
+          <span slot="labelB" style="color: gray"
+            >{$t('components.settings.customize_lms.enabled')}</span
+          >
+        </Toggle>
+      </div>
+    </Column>
+  </Row>
+
+  <div class="sticky desktop float-right bottom-12 mr-2 z-[120]">
+    <PrimaryButton
+      label={$t('components.settings.customize_lms.save')}
+      onClick={handleSave}
+      isLoading={isSaving}
+      isDisabled={isSaving}
+    />
+  </div>
+</Grid>
+
+<div
+  class="absolute
+   mobile right-6 bottom-8 z-[120]"
+>
+  <span>
+    <IconButton onClick={handleSave} disabled={isSaving}>
+      <Save size={40} class=" bg-blue-700 SectionTitle-1 rounded-full" />
+    </IconButton>
+  </span>
+</div>
+
+<style>
+  @media screen and (min-width: 769px) {
+    .desktop {
+      display: block;
+    }
+    .mobile {
+      display: none;
+    }
+  }
+  @media screen and (max-width: 768px) {
+    .desktop {
+      display: none;
+    }
+    .mobile {
+      display: flex;
+    }
+  }
+</style>
```

Comment:
There is no need for this cause you could simply use tailwind.

Instead of mobile you put a class in the element `block md:hidden` and instead of desktop you put a class `hidden md:block`